__tests__/testUtils.js

```diff
@@ -46,11 +46,7 @@ export function createInnerSlider({ noOfSlides, ...settings }) {
   const children = React.Children.toArray(
     createReactSliderChildren(noOfSlides)
   );
-  return (
-    <InnerSlider {...settings} ref={slider => (this.innerSlider = slider)}>
```

Comment:
hmm, this `ref` was causing the test failures with `TypeError: cannot set innerSlider of undefined`